Exercises/Chapter-4.txt

```diff
@@ -0,0 +1,44 @@
+1. Assign variables city and state to your current city and state of residence. (If residing outside the U.S., substitute the analogous quantities.)
+city = "Brooklyn"
+state = "New York"
+
+
+2. Using interpolation, print (using puts) a string consisting of the city and state separated by a comma and as space, as in “Los Angeles, CA”.
+puts "#{city}, #{state}"
+
+3. Repeat the previous exercise but with the city and state separated by a tab character.
+puts "#{city} v\ #{state}" (Joe's comment: I don't know what the fuck a tab charater is.)
```

Comment:
google "ascii tab"

it's a "non printable" character.  Special stuff like that is often typed with the format `\<something special>`.  Tab is `\t`.  This is called an "escape sequence".  The backslash "escapes" normal operation, so the "t" is interpreted as something special instead of the regular character t

It's old school and generally not used anymore.  Formatting (ie. moving the cursor to the next tab stop) is best done separate from content (ie. CSS for formatting, HTML for content)